demisto_sdk/commands/lint/resources/pylint_plugins/certified_partner_level_checker.py

```diff
@@ -151,9 +155,15 @@ def _demisto_results_checker(self, node):
         Adds the relevant error message using `add_message` function if one of the above exists.
         """
         try:
-            if node.func.attrname == "results" and node.func.expr.name == "demisto":
-                self.add_message("demisto-results-exists", node=node)
-
+            file_path = node.root().file.replace(".py", ".yml")
+            with open(file_path) as file_obj:
+                loaded_file_data = yaml.load(file_obj)  # type: ignore
```

Comment:
do not use blank `type:ignore`s, specify the error code.

Reply:
ah no, i did not intend to keep it, removing